_layouts/partners/faq.html

```diff
@@ -92,7 +92,7 @@ <h3 class="usa-accordion__heading">
                   </div>
                   <a href="#main-content">Back to Top</a>
                 </div>
-                <div class="tablet:grid-col-3 search-container">
+                <div class="tablet:grid-col-3">
```

Comment:
I was wondering whether we still need this class `.search-container` in the Sass code, and noticed it's also used on [the "Business inquiries" page](https://federalist-17bd62cc-77b7-4687-9c62-39b462ce6fd5.sites.pages.cloud.gov/preview/18f/identity-site/np/lg-7488/partners/business-inquiries/). Does that have the same visual glitch we're trying to address here?

![image](https://user-images.githubusercontent.com/1779930/200639105-082e1551-3c82-4126-a77e-0363b540a9bb.png)

Reply:
Good catch, yes it looks like its misaligned there too. I removed search-container from the sass and from that file.